ergo-lib/Cargo.toml

```diff
@@ -29,10 +29,12 @@ thiserror = "1"
 derive_more = "0.99"
 # bounded-vec = { git = "https://github.com/ergoplatform/bounded-vec", rev="f684d78" }
 bounded-vec = { version = "^0.5.0" }
-num-bigint = "0.4.0"
+num-bigint = { version = "0.4.0", features = ["serde"] }
 proptest-derive = {version = "0.3.0", optional = true }
 sha2 = { version = "0.9.8" }
 hmac = { version = "0.11.0" }
+k256 = { version = "0.10.0", features = ["arithmetic", "ecdsa"] }
+hex = "0.4.3"
```

Comment:
We're already using `base16` library for hex encoding. Do you have a reason to prefer `hex` over `base16`?

Reply:
After new changes, there is no need for `hex` anymore so I remove it